src/MoBi.Presentation/Tasks/DimensionValidator.cs

```diff
@@ -33,32 +36,32 @@ public DimensionValidator(IDimensionParser dimensionParser, IObjectPathFactory o
          _userSettings = userSettings;
          _hiddenNotifications = new Dictionary<string, string>
          {
-            {"Surface area (interstitial/intracellular)", "does not evaluate to the dimension 'Area'"},
```

Comment:
Always surprised to see some serious code duplication! Changing the constant in `AppConstants` (for example to make it sound more english by removing `the`) would break everything here